addons/event_sale/views/sale_order_views.xml

```diff
@@ -53,4 +53,16 @@
             </xpath>
         </field>
     </record>
+
+    <record id="sale_order_view_search_inherit_event_sale" model="ir.ui.view">
```

Comment:
Again, seems wildly overkill to have a separate primary mode view for an additional filter.

Let's just adapt the action domain to only show confirmed orders.

Reply:
The main purpose of adding a default filter is to allow users to still view unconfirmed orders there. (which I find quite helpful)

Reply:
I would agree if this filter was already available in the default view, but as it is not the case, let's not over complicate it.